src/usr/local/www/interfaces.php

```diff
@@ -781,6 +781,9 @@ function remove_bad_chars($string) {
 	if (($_POST['spoofmac'] && !is_macaddr($_POST['spoofmac']))) {
 		$input_errors[] = gettext("A valid MAC address must be specified.");
 	}
+	if (($_POST['dhcp6-duid'] && ($config['system']['global-v6duid'] == ""))) {
+		$input_errors[] = gettext("No System DUID has been entered.");
+	}
```

Comment:
I don't believe this should be treated as a fatal error. IMO only a warning should be OK